nx/lib/nx/lin_alg.ex

```diff
@@ -727,6 +727,40 @@ defmodule Nx.LinAlg do
         ]
       >
 
+      iex> {qs, rs} = Nx.LinAlg.qr(Nx.tensor([[[-3, 2, 1], [0, 1, 1], [0, 0, -1]],[[3, 2, 1], [0, 1, 1], [0, 0, 1]]]))
+      iex> qs
+      #Nx.Tensor<
+        f32[2][3][3]
+        [
+          [
+            [1.0, 0.0, 0.0],
+            [0.0, 1.0, 0.0],
+            [0.0, 0.0, 1.0]
+          ],
+          [
+            [1.0, 0.0, 0.0],
+            [0.0, 1.0, 0.0],
+            [0.0, 0.0, 1.0]
```

Comment:
Minor nitpick, but I'll change this later myself: I think it would be nice to show a matrix where one of the Q matrices is different from the other